pages/blog/june-2021-at-asyncapi.md

```diff
@@ -0,0 +1,187 @@
+---
+title: June 2021 at AsyncAPI
+date: 2021-07-09T06:00:00+01:00
+type: Communication
+tags:
+  - status
+  - news
+cover: /img/posts/june-2021-at-asyncapi/cover.webp
+authors:
+  - name: Lukasz Gornicki
+    photo: /img/avatars/lpgornicki.webp
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Community Guardian
+excerpt: 2.1.0 spec released, GSoC started, CLI released, other tools under heavy development. June was a hell of a ride.
+featured: true
+---
+
+> Read [May 2021 at AsyncAPI](/blog/may-2021-at-asyncapi) for the update from May.
+
+<iframe src="https://anchor.fm/asyncapi/embed/episodes/June-2021-at-AsyncAPI-Initiative-e144cup" height="102px" width="400px" frameborder="0" scrolling="no"></iframe>
+
+## Specification 2.1.0 release
+
+The eagle has landed. The new version of the AsyncAPI specification is here. For more details, read [AsyncAPI Spec 2.1.0 Release Notes](https://www.asyncapi.com/blog/release-notes-2.1.0).
+
+For more details on security improvements introduced into the spec by [Dale Lane](https://github.com/dalelane), read [this interactive article](https://dalelane.co.uk/blog/?p=4409).
+
+For more details on new properties in examples, check out the demo of the latest version of Microcks where [Laurent Broudoux](https://github.com/lbroudoux) shows them in action in the new release:
+
+<YouTube id="ksYrvrqj8NI" />
+
+A significant side effect of the release is that we automated the process of publishing the specification. Now [AsyncAPI Website](https://www.asyncapi.com/docs/specifications/v2.1.0) always reflects the latest version of specification document from [the spec repository](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md).
```

Comment:
Mention Aayush here maybe? He took part of it.

Reply:
sure, I've put special thanks in release notes already but why not doing it here :)